tests/integration/rules/length.phpt

```diff
@@ -24,8 +25,21 @@ try {
     echo $e->getMessage().PHP_EOL;
 }
 
+try {
+    v::length(5, 5)->check('123456');
+} catch (LengthException $e) {
```

Comment:
I'd rather not abbreviate variables, in this case, it would be `$exception` instead.